src/SdkCommon/AppAuthentication/Azure.Services.AppAuthentication.Unit.Tests/Mocks/MockMsi.cs

```diff
@@ -61,6 +64,27 @@ protected override Task<HttpResponseMessage> SendAsync(HttpRequestMessage reques
                     };
                     break;
                 case MsiTestType.MsiAppServicesSuccess:
```

Comment:
MsiAppServicesSuccess should be with MsiAzureVmSuccess. This is causing couple of tests to fail.